File: HackerRank/Python/Arrays.py
# Link --> https://www.hackerrank.com/challenges/30-arrays/problem

# Code:
if __name__ == '__main__':
    n = int(input().strip())
    arr = list(map(int, input().rstrip().split()))
    
    for i in range(len(arr)):
        print(arr[len(arr) - i - 1], end=" ")

File: HackerRank/JavaScript/Arrays.js
// Link --> https://www.hackerrank.com/challenges/js10-arrays/problem

// Code:
function getSecondLargest(nums) {
    let max = Math.max(...nums);

    let secondMax = 0; 
    for(let i=0; i<nums.length; i++) {
        if(nums[i] > secondMax && nums[i] < max)
            secondMax = nums[i];
    }
    return secondMax;
}
